components/usage/pkg/apiv1/billing.go

```diff
@@ -83,8 +83,14 @@ func (s *BillingService) GetStripeCustomer(ctx context.Context, req *v1.GetStrip
 			logger.WithError(err).Error("Failed to lookup stripe customer from DB")
 		}
 
+		stripeCustomer, err := s.stripeClient.GetCustomer(ctx, customer.StripeCustomerID)
```

Comment:
We're doing the fetch here only to grab the currency. Should we store it in the stripe_customer table instead to avoid this lookup? Otherwise we're again creating a source of truth in two places

Reply:
> only to grab the currency

Yeah, not really happy about that, indeed.

> Otherwise we're again creating a source of truth in two places

@easyCZ, about that. In the beginning Stripe was the source of truth, especially for everything related to the Customer. What has changed since? 
If we store it in the stripe_customer table, shall we stop putting it into Stripe?